randomStyle.js

```diff
@@ -0,0 +1,17 @@
+
+const h =document.querySelector('.height').value;
+h.setAttribute('value','height');
+
+const w =document.querySelector('.width').value;
+w.setAttribute('value','width');
+
+document.querySelector('#randomize').addEventListener('click', createImage);
+
+function myFunction() {
+const img = document.queryElement('.image');
+const url = 'https://source.unsplash.com/random/' + w +'/'+ h;
+img.setAttribute('src',url);
+
+const imgLocation = document.querySelector('.image');
+imgLocation.appendChild(newImage);
+}
```

Comment:
This how I will write the function
```suggestion
function myFunction() {
    const h =document.querySelector('.height').value;
    const w =document.querySelector('.width').value;
    const img = document.createElement('img');
    const url = 'https://source.unsplash.com/random/' + w +'x'+ h;
    img.setAttribute('src',url);
    
    const imgLocation = document.querySelector('.image');
    imgLocation.appendChild(img);
}
document.querySelector('#randomize').addEventListener('click', myFunction);
```